### DISCUSSION TITLE
th_to_CSV

### DISCUSSION THREAD

**Discussion by sashi1994** [OpenRadioss/Tools#10] [Q&A]
Is there any way that we can convert the source code of "th_to_csv.c" into .py file so I can extract data as I need?
Or if there is any python code that already exists that reads directly T01 that would be great. 
 (PS: I'm comfortable with python)

Thanks

**Comment by sebastienVilleneuve** [comment]
Hi @sashi1994 

A python code for reading th file is already existing in https://github.com/Vortex-CAE/Vortex-Radioss
Please request acces to this repo to https://github.com/Vortex-CAE


Sébastien
